pyproject.toml

```diff
@@ -111,7 +111,7 @@ filterwarnings = [
 ]
 log_level = "DEBUG"
 
-[tool.ruff]
+[tool.lint]
```

Comment:
Should this be `[tool.ruff.lint]`?

Reply:
Yep, you're right. I was following this confusing warning:

```
warning: The top-level linter settings are deprecated in favour of their counterparts in the `lint` section. Please update the following options in `pyproject.toml`:
  - 'select' -> 'lint.select'
  - 'isort' -> 'lint.isort'
```